plugins/ecs-bridge/engine/engine.go

```diff
@@ -143,6 +161,19 @@ func (engine *engine) createBridge(bridgeName string, mtu int) error {
 			"bridge create: unable to add bridge interface %s", bridgeName)
 	}
 
+	mac, err := generateMACAddress()
+	if err != nil {
+		return errors.Wrapf(err,
+			"bridge create: unable to generate mac addr %s", err)
+	}
+
+	log.Infof("Setting ecs-bridge hardware addr (MAC) %v %v", bridge, mac)
```

Comment:
Looking at command.go and engine.go, it seems the logging is done by command.go before calling the implementation of engine in engine.go. We may want to stick with that convention and remove the log statement here.

Reply:
I would prefer to keep this log statement. Is there any drawback to logging this? Historically we've generally had issues with these plugins under-logging and not telling us what they are doing, rather than the opposite.

Reply:
there's virtually no risk of this spamming the logfile, as this function is generally only called once per instance.

Reply:
We still log the error if the operation fails, correct?

Reply:
yes, but issues where there's an explicit failure are always the easiest to debug :)

Reply:
I still don't think this log statement provides any additional value, but don't have a strong opinion. Let's go with your approach.